_layouts/default.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en-us">
   <head>
     <meta charset="UTF-8">
-    <title>{{ page.title | default: site.title }}</title>
+    <title>{{ site.title | default: site.github.repository_name }} by {{ site.github.owner_name }}</title>
```

Comment:
No all the others I saw were:

`{{ site.title | default: site.github.repository_name }} by {{ site.github.owner_name }}`

one theme was missing by `{{ site.github.owner_name }}`.

The weird thing is if you don't specify `page.title` in the front matter Jekyll seems to pickup the text of the first heading and insert it into the title.